src/components/map/MapDirective.js

```diff
@@ -125,6 +124,60 @@ goog.require('ga_permalink');
                 gaOffline.hideExtent();
               }
             });
+
+            var savedTimeStr = {};
+            scope.$on('gaTimeChange', function(evt, time, oldTime) {
+              var switchTimeActive = (!oldTime && time);
+              var switchTimeDeactive = (oldTime && !time);
+              var olLayers = scope.map.getLayers().getArray();
+              var singleModif = false;
+
+              // Detection the time change has been triggered by a layer's
+              // 'propertychange' event.
```

Comment:
Each time you modify the time property of layer, we try to display/hide the time selector components if all the layers have the same time value or not.

Special case: when you modify the layer's time and then all the layer haven't the same time value anymore. 
In that specific case we don't want the new `undefined` value of the global time will be set to all the layers, we want the layers keep their last time value.